roles/network_scripts/molecule/shared/prepare.yml

```diff
@@ -0,0 +1,20 @@
+- name: register RHSM
+  hosts: all
+  tasks:
+    - name: create directory
+      file:
+        path: /tmp/network-scripts
```

Comment:
_tempfile_ module perhaps?

Reply:
This needs to be a known path in `playbook.yml` as this directory will be the destination for the test scripts